src/Symfony/Component/Uri/.gitattributes

```
@@ -0,0 +1,4 @@
+Tests export-ignore
+/phpunit.xml.dist export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
```

```suggestion
/.git* export-ignore
```
As Nicolas changed that recently